src/sage/numerical/pivot_rules_for_simplex_method.py

```diff
@@ -0,0 +1,776 @@
+r"""
+Pivot Rules for Simplex Method
+
+This module is an addendum to the Interactive Simplex Method
+to draw connection between active research on simplex method pivot rules
+and an educational settings.
+
+A pivot rule is a function or algorithm on an linear program (LP) `\min\{cx: Ax<=b\}`
+- (a) `v` is adjacent to `u` in the graph of $P$ and
+- (b) `cu \leq cv`.
+
+The module offers a number of predefined pivot rules and classes which serve as plug and play for defining/experimenting with pivot rules.
+
+Authors:
+
+-Acadia Larsen (March 1, 2024): Initial Version.
+
+EXAMPLES:
+
+This module is designed for use with :mod:`sage.interactive_simplex_method`::
+    sage: A = ([1, 1], [3, 1], [-1, -1])
```

Comment:
```suggestion
This module is designed for use with :mod:`sage.interactive_simplex_method`::

    sage: A = ([1, 1], [3, 1], [-1, -1])
```
Need blank line before the examples